Objects/dictobject.c

```diff
@@ -300,9 +300,18 @@ _PyDict_DebugMallocStats(FILE *out)
 
 static void free_keys_object(PyInterpreterState *interp, PyDictKeysObject *keys);
 
+/* We might consider modifying PyDictKeysObject so we could use
+   Py_INCREF() instead of dictkeys_incref() (and likewise with
+   Py_DECREF() and dictkeys_decref()).  However, there doesn't
+   seem to be enough benefit (performance or otherwise) to justify
+   the change. */
```

Comment:
I don't like this comment. Since dictkeys doesn't have type pointer, it is too trickey to use `Py_DECREF()`.

Reply:
I'll fix the comment.